docs/_includes/Configuring-Alluxio-with-GCS/underfs-address.md

```diff
@@ -1,3 +1,3 @@
 ```properties
-alluxio.underfs.address=gs://GCS_BUCKET/GCS_DIRECTORY
+alluxio.master.mount.table.root.ufs=gs://GCS_BUCKET/GCS_DIRECTORY
```

Comment:
would it be a good idea to rename these files from `underfs-address.md` to something like `root-mount-address.md`?

Reply:
@apc999 What's the future for these kind of files? I noticed that only a few of them are still linked in the codebase, is the intention to phase them out?

Reply:
yes, there are new contributor tasks to phase them out. In the future, we will not use code snippets md files but always put these inline in the md files

Reply:
@ZacBlanco Since we will be phasing the files out, are you okay with not modifying the name (at that point I would rather just in-line since I would need to modify all the internal references)

Reply:
Yes, I'm fine leaving it for now